modules/prebidServerBidAdapter/index.js

```diff
@@ -1107,6 +1114,8 @@ export function PrebidServer() {
           requestJson,
           { contentType: 'text/plain', withCredentials: true }
         );
+      } else {
+        utils.logError('Prebid Server request did not execute.  Please ensure endpoint URL(s) are properly defined in your s2sConfig.  Otherwise check for warnings/errors, likely noted earlier in console logs...');
```

Comment:
Again, with the goal of saving bytes in the package, how about something more terse like

"PBS request not made. Check endpoints."

Reply:
updated.